packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { FC, useState } from 'react';
+import { ItemRowFragment } from '@openmsupply-client/system';
```

Comment:
Then, I just updated the need to use the `Item` type in the `StocktakeLineEdit` modal to use the `ItemRowFragment` instead